src/common/utils/parseAttributes.ts

```diff
@@ -1,50 +1,49 @@
-const encodeAttribute = (attribute: string, jsonString: string): string => {
```

Comment:
[Note] 'attribute' was several layers up and was just ['details']. This is why I did this.